### PR TITLE
Issue #356 by Inlead: Implement environment indicator and configurations.

### PR DESCRIPTION
**Pull request by inlead**
http://platform.dandigbib.org/issues/356

### CODE REVIEW

File: ding2.install [hunk 1]
```diff
@@ -327,3 +306,10 @@ function ding2_update_7014() {
 function ding2_update_7015() {
   module_enable(array('search_api', 'search_api_multi', 'search_api_db', 'search_api_views'));
 }
+
+/**
+ * Enable search_api modules.
```
Comment (xendk) [inline ding2.install]: Fix comment.

File: ding2.install [hunk 2]
```diff
@@ -327,3 +306,10 @@ function ding2_update_7014() {
...
+ */
+function ding2_update_7016() {
+  module_enable(array('ddb_cp', 'environment_indicator'));
```
Comment (xendk) [inline ding2.install]: According to https://github.com/ding2/ding2/blob/master/modules/ddb_cp/README.md ddb_cp is only relevant for DBC hosted sites. Many sites are not.

File: modules/ddb_cp/ddb_cp.variable.inc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * @file
+ * ddb_cp.variable.inc
+ */
+
+/**
+ * Implements hook_variable_realm_default_variables().
+ */
+function ddb_cp_variable_realm_default_variables() {
```
Comment (xendk) [inline modules/ddb_cp/ddb_cp.variable.inc]: Why this?

**Comment by JrgenGNielsen** [comment]
reviewed and accepted

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: d97ac2f5ebfa4a584910839151043b7fdb4e8874
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: d62c823402ce0599bffcc2b364fe782bd0213d0e
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: c3375cab72811825b590bcdf84dd233d5db56c99
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: d97ac2f5ebfa4a584910839151043b7fdb4e8874
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: d62c823402ce0599bffcc2b364fe782bd0213d0e
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: c3375cab72811825b590bcdf84dd233d5db56c99
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by kasperg** [comment]
Reviewed and approved. Merged.

Next time please ensure that commit message formats are followed.

**Pull request merged**